package.json

```diff
@@ -9,6 +9,7 @@
     "test": "npm run build && nyc mocha --exit --timeout 20000 tests/*test.js ",
     "test:debug": "mocha --inspect-brk --exit tests/*test.js ",
     "coverage": "nyc report --reporter=text-lcov | coveralls",
+    "import-sort": "npx import-sort-cli --write '{src,examples}/**/*.ts{,x}'",
```

Comment:
I already added `prettier-plugin-organize-imports` this should sort the imports by default native typescript API
So we don't need a third party sorting anymore
Also this `import-sort` maybe don't supports TS 3.8 `import type` syntax

Reply:
It does not work with prettierx nor prettier.

Reply:
Ah I know what the problem is
We need to write something like `"format": "prettier --write . && prettierx --write ."`

Because prettierx is a little bit behind and doesn't support the installed plugin right now
But when running prettier before we have the correct ordering

Reply:
We need `"format": "prettier --loglevel silent --write . && prettierx --write ."`

Reply:
I also created an issue https://github.com/brodybits/prettierx/issues/463, so hopefully they support these plugins natively like prettier does